gitea/releases.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package gitea
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+// Release represents a repository release
+type Release struct {
+	ID           int64     `json:"id"`
+	TagName      string    `json:"string"`
+	Target       string    `json:"target_commitish"`
+	Title        string    `json:"title"`
+	Note         string    `json:"body"`
+	IsDraft      bool      `json:"draft"`
+	IsPrerelease bool      `json:"prerelease"`
+	Created      time.Time `json:"created"`
+	Publisher    *User     `json:"author"`
+}
```

Comment:
I've added `url`, `tarball_url`, `zipball_url`, `name`, and `published_at`. I did not add `html_url`, `assets_url`, or `upload_url`, since they don't currently apply to Gitea.